backend/core/models/step.py

```diff
@@ -15,8 +15,7 @@ class Step(CommonInfo, SoftDeleteObject):
     id = models.AutoField(primary_key=True)
     text = models.TextField()
     recipe = models.ForeignKey["Recipe"]("Recipe", on_delete=models.CASCADE)
-    if TYPE_CHECKING:
-        recipe_id: int
+    recipe_id: int
```

Comment:
this isn't going to overwrite the Django property?

Reply:
I think it's safe, since it doesn't actually assign anything, it's just a declaration 

```python
In [1]: class Foo:
   ...:     x: int
   ...: 

In [2]: Foo().x
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Input In [2], in <cell line: 1>()
----> 1 Foo().x

AttributeError: 'Foo' object has no attribute 'x'

In [3]: 
```